src/content/en/2022/mobile-web.md

```diff
@@ -2,21 +2,436 @@
+
+{{ figure_markup(
+    image="good-lcp.png",
+    caption="Annual comparison of the percent of websites having good LCP on mobile.",
+    description="Bar chart showing 43% of websites having good LCP in 2020, 45% in 2021, and 51% in 2022.",
+    chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQ2iu1w8TLfOTLCQ1O-d74ku9NirGCQJNeO789uKX1-M6N053ASsz2tXWPvU1twhScsa3rxWF7_7OBn/pubchart?oid=1991240853&format=interactive",
+    sheets_gid="381713804",
+    sql_file="web_vitals_by_device.sql"
+  )
+}}
+
+LCP performance is improving. In 2020, 43% of mobile sites had a LCP assessed as “good”. In 2021, this number improved to 45% of mobile sites. There was a significant jump in 2022 in which 51% of mobile sites had good LCP performance. The [Performance](../performance) chapter explores some possible explanations for why this may have happened.
+
+
+### Images
+
+Images can contribute a lot to providing a good mobile experience, but they can also contribute a lot to slow performance and bad loading experiences if they are not set up correctly. This section explores how site owners are handling—or not handling—the performance impact of images.
+
+
+#### Appropriately sized images
+
+Using images that are sized properly for a mobile device has long been one of the easiest ways that anyone could improve mobile load time. In the early days of the mobile web, site owners would often simply send the same images to desktop users as they would for mobile users, because ultimately, mobile browsers would scale and resize the images to fit in the mobile rendering of the page. Unfortunately, this didn’t work well, because it ended up requiring a lot of extra data to send rich, high-quality images that were better suited for a desktop viewing experience.
+
+{{ figure_markup(
+    image="appropriately-sized-images.png",
+    caption="Annual comparison of the percent of websites that have appropriately sized images.",
+    description="Bar chart showing 59% of websites having properly sized images in 2020, 57% in 2021, and 54% in 2022.",
+    chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQ2iu1w8TLfOTLCQ1O-d74ku9NirGCQJNeO789uKX1-M6N053ASsz2tXWPvU1twhScsa3rxWF7_7OBn/pubchart?oid=1739495492&format=interactive",
+    sheets_gid="1709663847",
+    sql_file="correctly_sized_images.sql"
+  )
+}}
+
+Given Google’s increased focus on rewarding good performance with their Core Web Vitals program, you would expect that more sites would be optimizing their images. However, it’s interesting to see that sites are actually having fewer optimized images over time. The figure above shows that there is a decrease in the percentage of pages with properly sized images since [2020](https://almanac.httparchive.org/en/2020/mobile-web#images), when 59% of sites had properly sized images. But in 2022, that number is down to only 54%.
```

Comment:
```suggestion
Given Google’s increased focus on rewarding good performance with their Core Web Vitals program, you would expect that more sites would be optimizing their images. However, it’s interesting to see that sites are actually having fewer optimized images over time. The figure above shows that there is a decrease in the percentage of pages with properly sized images since [2020](../2020/mobile-web#images), when 59% of sites had properly sized images. But in 2022, that number is down to only 54%.
```